packages/salesforcedx-vscode-core/src/panels/orgListPanel.ts

```diff
@@ -155,7 +154,7 @@ export class OrgListPanel {
 
                 <vscode-data-grid
                   class='all-scratch-orgs-data-grid'
-                  grid-template-columns='240px 320px 200px 200px 200px 200px'
+                  grid-template-columns='50px 1fr 2fr 1fr 1fr'
```

Comment:
One of the columns is missing, and should probably be `'50px 1fr 2fr 1fr 1fr 1fr'`

However, even with the extra column, the alignment is off.  What I see in the "All" tab is:
![image](https://user-images.githubusercontent.com/46358907/175440749-40b0228f-eca1-4b19-8470-a116855cf677.png)